### PR TITLE
Additional branch tests covering test plan 11.1 and 11.5

### PR DESCRIPTION
**Pull request by mgibbs1259**
## Pull Request for Issue #63

**Description of Changes**
Created two tests covering test plan 11.1 and 11.5
11.1 - Files created in workspace, deleted in feature branch, and then merged back into workspace should NOT show up in workspace
11.5 - Files favorited in a feature branch should be favorites when that branch is merged back into “workspace”

**Exit Criteria**
Tested on Chrome and Firefox

### CODE REVIEW

File: gigantum_tests/test_branching.py [hunk 1]
```diff
@@ -13,26 +14,97 @@ def test_create_local_branch(driver: selenium.webdriver, *args, **kwargs):
...
+    time.sleep(2)
+    branch_elts.switch_to_master_branch()
+    branch_elts.merge_test_branch_into_master_branch()
+    logging.info(f"Checking that file deleted in test-branch does not appear in master branch")
+
+    assert file_browser_elts.file_browser_empty.find(), \
+        "Expected sample-upload.txt to not appear in master branch"
+
+
+def test_favorite_file_local_branch(driver: selenium.webdriver, *args, **kwargs):
```
Comment (billvb) [inline gigantum_tests/test_branching.py]: Remove or comment out this test

File: gigantum_tests/test_branching.py [hunk 2]
```diff
@@ -13,26 +14,97 @@ def test_create_local_branch(driver: selenium.webdriver, *args, **kwargs):
...
+    file_browser_elts.favorite_file_button_off.find().click()
+    time.sleep(2)
+    branch_elts.switch_to_master_branch()
+    branch_elts.merge_test_branch_into_master_branch()
+    logging.info(f"Checking that file favorited in test-branch is favorited in master branch")
+
+    assert file_browser_elts.favorite_file_button_on.find(), \
+        "Expected sample-upload.txt to be favorited in master branch"
+
+
```
Comment (billvb) [inline gigantum_tests/test_branching.py]: There should be exactly one blank line at the end of the file :)

File: testutils/elements.py [hunk 3]
```diff
@@ -416,12 +425,40 @@ def manage_branches_branch_name(self):
...
+    def manage_branches_confirm_merge_branch_button(self):
+        return CssElement(self.driver, ".Branches__Modal-confirm")
+
     def create_local_branch(self, branch_name):
         logging.info("Creating a new local branch")
         self.create_branch_button.wait().click()
         self.branch_name_input.find().send_keys(branch_name)
         self.create_button.wait().click()
 
+    def switch_to_master_branch(self):
```
Comment (billvb) [inline testutils/elements.py]: Rename, maybe to like `switch_to_alternate_branch` and then explain what that means in a coment

File: gigantum_tests/test_branching.py (hunk 1 above)
Comment (mgibbs1259) [reply gigantum_tests/test_branching.py]: Commented out

File: gigantum_tests/test_branching.py (hunk 2 above)
Comment (mgibbs1259) [reply gigantum_tests/test_branching.py]: Oops... Fixed this!

File: gigantum_tests/test_branching.py [hunk 4]
```diff
@@ -13,26 +14,81 @@ def test_create_local_branch(driver: selenium.webdriver, *args, **kwargs):
...
-    owner, proj_name = graphql.create_py3_minimal_project(testutils.unique_project_name())
-    driver.get(f"{os.environ['GIGANTUM_HOST']}/projects/{owner}/{proj_name}")
-    time.sleep(4)
+    r = testutils.prep_py3_minimal_base(driver)
+    username, project_title = r.username, r.project_name
     branch_elts = testutils.BranchElements(driver)
-    branch_elts.create_local_branch("my-test-branch")
-    time.sleep(4)
+    branch_elts.create_local_branch("test-branch")
+    time.sleep(8)
```
Comment (billvb) [inline gigantum_tests/test_branching.py]: why a sleep and then a `find()`...

why not just a wait(10) or something?

File: gigantum_tests/test_branching.py [hunk 5]
```diff
@@ -13,26 +14,81 @@ def test_create_local_branch(driver: selenium.webdriver, *args, **kwargs):
...
+    r = testutils.prep_py3_minimal_base(driver)
+    username, project_title = r.username, r.project_name
+    logging.info(f"Navigating to Input Data")
+    driver.get(f"{os.environ['GIGANTUM_HOST']}/projects/{username}/{project_title}/inputData")
+    time.sleep(2)
+    logging.info(f"Adding a file to the master branch of project {project_title}")
+    file_browser_elts = testutils.FileBrowserElements(driver)
+    file_browser_elts.drag_drop_file_in_drop_zone()
+    time.sleep(4)
+    branch_elts = testutils.BranchElements(driver)
```
Comment (billvb) [inline gigantum_tests/test_branching.py]: Same thing here, why the sleep and finds rather than waits?

**Pull request merged**